datadog_checks_downloader/pyproject.toml

```diff
@@ -37,6 +37,7 @@ deps = [
     "securesystemslib[crypto,pynacl]==0.20.1",
     "tuf==0.17.0; python_version < '3.0'",
     "tuf==0.19.0; python_version > '3.0'",
+    "setuptools",
```

Comment:
Shouldn't we pin the version?

Reply:
I'm not sure if there should be pinned a specific version. What is the expected version range here?

Reply:
Yes, we should pin the version. The current latest version, for example.